fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -553,6 +553,7 @@ private CompletableFuture<Void> loadFileReferenceCacheForMemoization() {
             list.forEach(kv -> {
                 String name = metaSubspace.unpack(kv.getKey()).getString(0);
                 final FDBLuceneFileReference fileReference = Objects.requireNonNull(FDBLuceneFileReference.parseFromBytes(LuceneSerializer.decode(kv.getValue())));
+                agilityContext.recordSize(LuceneEvents.SizeEvents.LUCENE_FILES_COUNT, kv.getValue().length);
```

Comment:
This doesn't look right, this would record the size of each of the file references, which is not what we want, we want to record the size of `list`

Reply:
changed that!